recipes/quazip/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "quazip" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stachenov/quazip/archive/refs/tags/v1.1.tar.gz
```

Comment:
Not using `{{ version }}` in the URL here. Would be better for the autotick bot to do its magic.

Reply:
Done in https://github.com/conda-forge/staged-recipes/pull/16073/commits/ea0f2dccd3b05066ceba40ae935ffbd3c85f29a2 , thanks!